examples/release_highlights/plot_release_highlights_1_0.py

```diff
@@ -0,0 +1,180 @@
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 1.0! The library has
+been stable for quite some time, releasing version 1.0 is recognizing that and
+signalling it to our users. The release does not include any braking changes,
+and as usual, and we do our best to follow a two release deprecation cycle for
+any breaking changes.
+
+This release includes many bug fixes and improvements, as well as some new key
+features. We detail below a few of the major features of this release. **For an
+exhaustive list of all the changes**, please refer to the :ref:`release notes
+<changes_1_0>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install -c conda-forge scikit-learn
+"""
+
+##############################################################################
+# keyword and positional arguments
+# ---------------------------------------------------------
+# The scikit-learn API exposes many functions and methods which have many input
+# parameters. For example, before this release, one could instantiate a
+# `ensemble.HistGradientBoostingRegressor` as::
+#
+#         HistGradientBoostingRegressor("squared_error", 0.1, 100, 31, None,
+#             20, 0.0, 255, None, None, False, "auto", "loss", 0.1, 10, 1e-7,
+#             0, None)
+#
+# Understanding the above code requires the reader to go to the API
+# documentation to check each parameter, and what they mean, with the position
+# they have. To improve the readability of code written based on scikit-learn,
+# now users have to provide most parameters with their names, as keyword
+# arguments, instead of positional arguments. For example, the above code would
+# be::
+#
+#     HistGradientBoostingRegressor(
+#         loss="squared_error",
+#         learning_rate=0.1,
+#         max_iter=100,
+#         max_leaf_nodes=31,
+#         max_depth=None,
+#         min_samples_leaf=20,
+#         l2_regularization=0.0,
+#         max_bins=255,
+#         categorical_features=None,
+#         monotonic_cst=None,
+#         warm_start=False,
+#         early_stopping="auto",
+#         scoring="loss",
+#         validation_fraction=0.1,
+#         n_iter_no_change=10,
+#         tol=1e-7,
+#         verbose=0,
+#         random_state=None,
+#     )
+#
+# which is much more readable.
+
+##############################################################################
+# Spline Transformers
+# ---------------------------------------------------------
+# One way to add nonlinear terms the dataset's feature set is to generate
+# spline basis functions for each feature with the
+# :class:`preprocessing.SplineTransformer`. Splines are piecewise polynomials,
+# parametrized by their polynomial degree and the positions of the knots. The
+# :class:`preprocessingSplineTransformer` implements a B-spline basis
+
+# .. figure:: ../linear_model/images/sphx_glr_plot_polynomial_interpolation_001.png
```

Comment:
Looks like this is treated as a comment and not as part of the html group above, so the image isn't shown. I think the link above just misses a `#`?

Reply:
It now works, kinda, but seems like I have the wrong link, do you know how I can fix that @NicolasHug ?

Reply:
rendered version: https://151807-843222-gh.circle-artifacts.com/0/doc/auto_examples/release_highlights/plot_release_highlights_1_0.html

Reply:
It's possible that the other examples aren't run so the plots don't exist in the build
Maybe add a `[doc build]` commit to make sure?